modules/apps/foundation/frontend-theme/space-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,255 @@
+/* These inject tags are used for dynamically creating imports for themelet styles, you can place them where ever you like in this file. */
+
+/* inject:imports */
+/* endinject */
+
+/* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
+/*
+    ------------------------
+    ----- 1. Variables -----
+    ------------------------
+*/
+// @import "_navbar";
+//Liferay Media Queries 
+$screen-lg: 1200px !default;
+$screen-lg-min: $screen-lg;
+$screen-lg-desktop: $screen-lg-min;
+
+$screen-md: 992px !default;
+$screen-md-min: $screen-md;
+$screen-desktop: $screen-md-min;
+
+$screen-sm: 768px !default;
+$screen-sm-min: $screen-sm;
+$screen-tablet: $screen-sm-min;
+
+$screen-xs: 480px !default;
+$screen-xs-min: $screen-xs;
+$screen-phone: $screen-xs-min;
+
+$screen-md-max: ($screen-lg-min - 1) !default;
+$screen-sm-max: ($screen-md-min - 1) !default;
+$screen-xs-max: ($screen-sm-min - 1) !default;
+
+$media-query-sizes: (
+	xs: $screen-xs,
+	sm: $screen-sm,
+	md: $screen-md,
+	lg: $screen-lg
+);
+
+
+
+
+$background: #333b43;
+$background-alt: #f2f2f5;
+$base-color: #0272aa;
+$base-color-hover: darken($base-color, 10%);
+$blue: #3c4251;
+$dark-blue: #333745;
+$grey: #f5f2f5;
+$heading: #374054;
+$light-blue: #67aae9;
+$off-black: #20222b;  
+$off-white: #ffffff; 
+$text: #74808a;
+$theme: #20222b;
+
+
+$xxl-size: 6.2rem;
+$xl-size: 4.8rem;
+$l-size: 3.2rem;
+$m-size: 1.6rem;
+$s-size: 1.2rem;
+$desktop-breakpoint: 45rem;
+
+
+#wrapper {
+
+    margin: 0 auto;
+    padding:0;
+    position:relative;
+    background-color: $background;
+    color: $off-black;
+
+        h1 {
+            font-size: $xxl-size;
+            font-family: Helvetica, Arial;
+            font-weight: 1000;
+            color: $off-black;
+            width: 300px;
+            display: inline-block;
+            vertical-align: middle;
+            margin-top: 10px;
+            margin-bottom: 10px;
+
+        }
+
+        #heading {
+            background-color: $off-white;
+        }
+
+        #navigation {
+            background-color: $background;
+            height: 100px;
+
+            .container-fluid {
+                padding-left: 37px;
+                padding-right: 37px;
+                background-color: $off-white;
+                position: fixed;
+                width: 100%;
+                z-index: 1000;
```

Comment:
Make sure your ```z-indexes``` match the values defined in https://github.com/liferay/liferay-portal/blob/master/modules/apps/foundation/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js#L1876...L1888.